modelmesh-runtime-adapter.yaml

```diff
@@ -0,0 +1,130 @@
+package:
+  name: modelmesh-runtime-adapter
+  version: 0.12.0
+  epoch: 0
+  description: Unified runtime-adapter package of the sidecar containers which run in the modelmesh pods
+  dependencies:
+    runtime:
+      - py3-absl-py
+      - py3-asn1
+      - py3-asn1-modules
+      - py3-astunparse
+      - py3-cachetools
+      - py3-certifi
+      - py3-charset-normalizer
+      - py3-flatbuffers
+      - py3-gast
+      - py3-google-auth
+      - py3-google-auth-oauthlib
+      - py3-google-pasta
+      - py3-grpcio
+      - py3-h5py
+      - py3-idna
+      - py3-importlib-metadata
+      - py3-keras
+      - py3-libclang
+      - py3-markdown
+      - py3-markupsafe
+      - py3-numpy
+      - py3-oauthlib
+      - py3-opt-einsum
+      - py3-packaging
+      - py3-pip
+      - py3-protobuf
+      - py3-requests
+      - py3-requests-oauthlib
+      - py3-rsa
+      - py3-setuptools
+      - py3-six
+      - py3-tensorflow-core
+      - py3-tensorflow-io-gcs-filesystem
+      - py3-termcolor
+      - py3-typing-extensions
+      - py3-urllib3
+      - py3-werkzeug
+      - py3-wheel
+      - py3-wrapt
+      - py3-zipp
+      - python-3
+
+data:
+  - name: binaries
+    items:
+      mlserver-adapter: model-mesh-mlserver-adapter
+      ovms-adapter: model-mesh-ovms-adapter
+      torchserve-adapter: model-mesh-torchserve-adapter
+      triton-adapter: model-mesh-triton-adapter
+      puller: model-serving-puller
+
+# Wheels for tensorboard, tensorboard-data-server, tensorflow-estimator
```

Comment:
wondering if we can also package this as `py3-*` and include this as runtime dependencies.